modules/fundamental/src/purl/endpoints/mod.rs

```diff
@@ -68,38 +65,6 @@ pub async fn get(
     }
 }
 
-#[utoipa::path(
-    operation_id = "getPurls",
-    tag = "purl",
-    params(
-        Deprecation
-    ),
-    request_body = PurlsRequest,
-    responses(
-        (status = 200, description = "Details for the qualified PURLs", body = PurlsResponse),
-    ),
-)]
-#[post("/v2/purl")]
-/// Retrieve details for multiple qualified PURLs
-pub async fn get_multiple(
-    service: web::Data<PurlService>,
-    db: web::Data<Database>,
-    request: web::Json<PurlsRequest>,
-    web::Query(Deprecation { deprecated }): web::Query<Deprecation>,
-    _: Require<ReadSbom>,
-) -> actix_web::Result<impl Responder> {
-    let items: Vec<&str> = request.items.iter().map(|s| s.as_str()).collect();
-    match service
-        .fetch_purl_details(&items, deprecated, db.as_ref())
```

Comment:
So we went to a lot of effort to change `fetch_purl_details` to take a list only to pass a single element to it all the time, going forward. If I'm looking at this code 6 months from now, I'm probably going to wonder why, especially if there's any performance impact to a sql 'IN' clause over an `=`.

Reply:
Agree, it's better to revert the whole PR. I'll open a separate one for that.

Reply:
That's fine, but wouldn't it make more sense to delete your old multi-purl endpoint in that separate PR, too?

Reply:
Indeed. That was my intention. To revert all the changes. Will do it ASAP

Reply:
See https://github.com/trustification/trustify/pull/1462